templates/services/merge/validator/lodestar.tmpl

```diff
@@ -30,9 +30,14 @@
   validator:
     container_name: validator-client
     image: ${VL_IMAGE_VERSION}
+    restart: unless-stopped
     depends_on:
       validator-import:
         condition: service_completed_successfully
+      validator-blocker:
+        condition: service_completed_successfully{{if .WithConsensusClient}}
```

Comment:
I don't see a field called `WithConsensusClient` within `DockerComposeData` struct. This shouldn't work as far as I see. The `Services` slice is there but is not being used.

Reply:
`WithConsensusClient` isn't a field, is a method that verifies that `Services` slice contained the "consensus" value, that is why it seems that it is not used